lib/puppet_x/puppetlabs/aws.rb

```diff
@@ -162,6 +162,14 @@ def elb_client(region = default_region)
         self.class.elb_client(region)
       end
 
+      def self.elbv2_client(region = default_region)
```

Comment:
For my own understanding, what is in the v2 client that we are using that is not present in the ELB client already in the code?